job_controller/test_job_controller.go

```diff
@@ -10,9 +10,9 @@ import (
 )
 
 type TestJobController struct {
-	job            *testv1.TestJob
-	pods           []*corev1.Pod
-	services       []*corev1.Service
+	job      *testv1.TestJob
+	pods     []*corev1.Pod
+	services []*corev1.Service
 }
```

Comment:
Sure, but probably in a follow-up PR to avoid conflicts with other people's work. Also, just wanted to double check with @jian-he since he added this file initially - is there a reason for putting this file under job_controller?

Reply:
If I remember correctly, I tried to put in the test_job folder, but couldn't do it due to circle dependency.